pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/TestParticipantUtil.java

```diff
@@ -68,27 +68,32 @@ public static void deleteParticipantAndInstitution(int participantId) {
         }
     }
 
-    public static ParticipantDto createSharedLearningParticipant(String guid, DDPInstanceDto ddpInstanceDto, String dob, String esIndex) {
+    private static ParticipantDto createParticipantFromConfigFiles(String guid, DDPInstanceDto ddpInstanceDto, String dob,
```

Comment:
@denniscunningham how's this looking?  Factored out a shared sub-method, but left existing method names intact since they are informative.